src/components/Toolbar.tsx

```diff
@@ -127,6 +130,11 @@ const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
   const onTapUp = useCallback(
     (id: ShortcutId) => {
       deselectPressingToolbarId()
+      if (id === 'newThought') {
+        store.dispatch(showTip({ tip: TipId.NewThought }))
+      } else if (id === 'newSubthought') {
+        store.dispatch(showTip({ tip: TipId.NewSubthought }))
+      }
```

Comment:
We'll just need to wrap this in `if (!customize) { ... }` to prevent the tip from being activated on the Customize Toolbar screen.

Reply:
Done: fdce1c02ff641d612852d6e172b1d3d096cb44ea